apps/challenges/aws_utils.py

```diff
@@ -0,0 +1,301 @@
+import os
```

Comment:
Is this the right location to store this file? @RishabhJain2018  can you confirm?

Reply:
@RishabhJain2018

Reply:
yes.